.github/workflows/api.yml

```
@@ -0,0 +1,36 @@
+name: Update API Data
+
+on:
+  push:
+    paths:
+    - 'keyboards/**'
+    - 'layouts/community/**'
+
+jobs:
+  api_data:
+    runs-on: ubuntu-latest
+    container: qmkfm/base_container
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Unless you want to have the whole repo here for history, you can limit how much you pull down
```suggestion
          fetch-depth: 1
```